docs/home/api-info/webhooks.mdx

```diff
@@ -36,6 +37,16 @@ different messages when they become available - specifically:
 1. If using the Devices API - accept and process [user data messages](#devices-api-messages).
 1. If using the Medical API - accept and process [patient data messages](#medical-api-messages).
 
+## Signature Verification
+
+When Metriport sends a webhook event, it includes an `x-metriport-signature` header. 
+
+- `x-metriport-signature`: This is a [HMAC](https://en.wikipedia.org/wiki/HMAC) SHA256 hash computed using your webhook key and the body of the webhook event.
+
+You can use this header to verify that the webhook events sent to your endpoint are from Metriport. Here's an example of how you can do this in Node.js:
+
+<Snippet file="webhook-signature-validation.mdx" />
+
```

Comment:
From the perspective of the developer, I'd be confused reading this guide - we have "Signature Verification" via HMAC, but then "Authentication" below via `x-webhook-key`.

Let's change this so that we have one "Authentication" section with the new HMAC method, and put the old `x-webhook-key` stuff in a deprecated section with a deprecation warning/timeline for devs to be aware.

Otherwise, we'll just have devs picking whatever system is more convenient for them, even though the old way is planned to be deprecated.

Reply:
Good catch!

I'd just remove the old approach, not even mention that it was an option. :)